src/Error/Error.php

```diff
@@ -121,7 +122,7 @@ public function __construct(
         if ($previous instanceof ClientAware) {
             $this->isClientSafe = $previous->isClientSafe();
             $cat                = $previous->getCategory();
-            $this->category     = $cat === '' || $cat === null  ? self::CATEGORY_INTERNAL: $cat;
+            $this->category     = $cat === '' || $cat === null  ? self::CATEGORY_INTERNAL : $cat;
```

Comment:
Oh how I wish multiline ternary expressions could be automated.

```suggestion
            $this->category     = $cat === '' || $cat === null
                ? self::CATEGORY_INTERNAL
                : $cat;
```

Reply:
same. But that's out of scope of this PR :P